src/Data/Integer/Properties.agda

```diff
@@ -1404,3 +1403,11 @@ Please use neg-involutive instead."
 "Warning: +-⊖-left-cancel was deprecated in v0.15.
 Please use +-cancelˡ-⊖ instead."
 #-}
+
+-- Version 0.18
+
+≰→> = ≰⇒>
+{-# WARNING_ON_USAGE ≰→>
+"Warning: ≰→> was deprecated in v0.15.
```

Comment:
Shouldn't this be v0.18?

Reply:
Eek, yes. Didn't do a very good job of this!